docs/stencil-docs/page-builder/product-widget-tutorial.md

```diff
@@ -73,65 +57,146 @@ Accept: application/json
   "template": "<div style=\"text-align:center\">\n<h1>{{_.data.site.product.name}}</h1>\n<div>\n<img src=\"{{_.data.site.product.defaultImage.url}}\">\n</div>\n<div>\n<p>${{_.data.site.product.prices.price.value}}</p>\n</div>\n</div>"
 }
 ```
+
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner)
 
+**[Response:](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#responses)**
+
+```json
+{
+  "data": {
+    "uuid": "479e6ffe-73bc-4a0c-8e49-654d97691e8b",
+    "name": "Product Widget",
+    ...
+}
+```
+
+|Property|Type|Description|
+|-|-|-|
+|`name`|string|Name of the widget|
+|`template`|string|The [widget template](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#widget-templates)|
+|`schema`|object|The widget settings [schema](https://developer.bigcommerce.com/stencil-docs/page-builder/widget-ui-schema) for [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder) UI|
```

Comment:
```suggestion
|`schema`|object|The customizable widget settings [schema](https://developer.bigcommerce.com/stencil-docs/page-builder/widget-ui-schema) for [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder) UI.|
```

Reply:
Are there widgets that aren't customizable?

Reply:
I reckon that all BigCommerce widgets are customizable. We can remove all reference to "custom" and "customizable" from this tutorial.